### PR TITLE
Mobile menu  popup  through Pair Programming

### PR DESCRIPTION
**Pull request by ometman**
Index.html was modified to add mobile menu navigation
The CSS files were modified to style the mobile menu
A Javascript file was added to support the opening of the menu
A close button was added to support the closing of the menu
The menu items were linked appropriately to the various sections when clicked
-Javascript linters files were added to enable testing and validation of js code.
-The README.md was updated to reflect contributors, future features, and project information.
-The menu was tested and was working
-Linters were run and passed

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -22,12 +22,12 @@
...
-                <input type="checkbox" class="mobile-menu-nav-toggle" id="mobile-menu-nav-toggle">
+                 <input type="checkbox" class="mobile-menu-nav-toggle" id="mobile-menu-nav-toggle">
                 <nav id="mobile-menu-nav" class="mobile-menu-nav">
                     <ul class="mobile-menu">
-                        <li><a href="#mob-works-section">Portfolio</a></li>
-                        <li><a href="#about-me-section">About</a></li>
-                        <li><a href="#contact-me-section">Contact</a></li>
+                        <li><a id="mobile-port" href="#mob-works-section">Portfolio</a></li>
+                        <li><a id="mobile-about" href="#about-me-section">About</a></li>
+                        <li><a id="mobile-contact" href="#contact-me-section">Contact</a></li>
```
Comment (andgarzonmal) [inline index.html:30]: - [ ] Please take into account that when the user clicks any of the displayed options in the mobile menu the menu should get closed as well. please take a look at this loom video showing you what i mean 

https://www.loom.com/share/45f0349f645048d8ad17e79febb4d1c6

**Review by andgarzonmal — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 Hi @ometman ✋🏼 

#### Your project is excellent. You are almost done. However, there are some issues you should fix. Please fix them and your project will be approved.

___ 

### 💯  Highlights 💯
- Your gitHub flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️

### ♻️ Request changes ♻️

- [x] _Check the comments under the review._
___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Comment by ometman** [comment]
Thank you. I guess that part was not committed when I noticed and fixed it. Thanks for the video work too. I will fix and commit it asap.

**Comment by GzimAsani** [comment]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- NA
### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**